deployments/aws/single-connector/main.tf

```diff
@@ -7,6 +7,22 @@
 
 locals {
   prefix = var.prefix != "" ? "${var.prefix}-" : ""
+  bucket_name = "${local.prefix}pcoip-scripts-${random_id.bucket-name.hex}"
+}
+
+resource "random_id" "bucket-name" {
+  byte_length = 3
+}
+
+resource "aws_s3_bucket" "scripts" {
+  bucket        = local.bucket_name
+  region        = var.aws_region
+  acl           = "private"
+  force_destroy = true
+
+  tags = {
+    Name = local.bucket_name
+  }
```

Comment:
This block is not present in the GCP bucket resource.

Reply:
GCP and AWS are different resources and have different properties